src/app/pages/Game.tsx

```diff
@@ -40,6 +40,10 @@ export const Game = () => {
 
 	const [blockScroll, allowScroll] = useScrollBlock()
 	const [activeRobot, setActiveRobot] = useState<string>()
+	const [fightUnlocked, setFightUnlocked] = useState<boolean>(false)
+
+	// Used to test if I could disable the inputs in robotconfig
+	const [readyToMove, setReadyToMove] = useState<boolean>(true)
```

Comment:
Should be derived from the GameEngine state.

Reply:
Not seeing how I can disable the fightUnlocked, as this communicates between the switch and the button

Reply:
Yes, correct. My bad.